.github/workflows/preprocess-gisaid.yml

```diff
@@ -0,0 +1,60 @@
+name: preprocess-gisaid
+
+on:
+  ## This workflow is intended to be run via a repository dispatch event sent from
+  ## https://github.com/nextstrain/ncov-ingest/blob/master/.github/workflows/fetch-and-ingest-gisaid-master.yml
+  repository_dispatch:
+    types:
+      - preprocess-gisaid
+  ## Manual running (via GitHub UI) is also be available, if needed, via workflow dispatch
+  workflow_dispatch:
+    inputs:
+      trial_name:
+        description: 'Short name to use as a prefix for the uploaded data. WARNING: without this we will overwrite the files in s3://nextstrain-ncov-private.'
+        required: false
+
+env:
+  S3_DST_BUCKET: ${{ github.event.inputs.trial_name && format('nextstrain-ncov-private/trial-preprocess/{0}', github.event.inputs.trial_name) || 'nextstrain-ncov-private' }}
```

Comment:
Why uploading to `trial-preprocess/…` instead of `trial/…` like the existing trial build destinations? Seems cleaner to keep the namespacing the same, particularly because it seems like it'd be helpful to optionally start some trial builds from the outputs of a trial preprocessing run of the same name ([rough, incomplete sketch of what I mean](https://gist.github.com/tsibley/e79bb12714acfbaecc7f3768243c7fac)).